android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -23,12 +21,106 @@
     private int       currentWidthPx;
     private int       currentHeightPx;
 
+    private String      adUnitId;
+    private MaxAdFormat adFormat;
+    private String      placement;
+    private String      customData;
+    private Boolean     adaptiveBannerEnabled;
+    private Boolean     autoRefresh;
```

Comment:
Why are we using `Boolean` objects for `adaptiveBannerEnabled` and `autoRefresh`? Should just be the `boolean` primitive type.

Reply:
Yes, updated to be the `boolean` primitive type.